go.mod

```diff
@@ -145,9 +144,6 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/batchpersi
 // This is a temporary solution while a new configuration design is discussed for the collector (ref: https://github.com/open-telemetry/opentelemetry-collector/issues/3482).
 replace go.opentelemetry.io/collector => github.com/grafana/opentelemetry-collector v0.4.1-0.20211008102704-a5d23001cb71
 
-// Pin prometheus dependencies
-replace github.com/prometheus/common => github.com/prometheus/common v0.23.0
```

Comment:
🎉 Happy to see that this pin can be removed now. 
Can we add one entry each in the CHANGELOG for "primary" dependencies that we updated? Anything in depcheck.yaml should be included.

Reply:
@rfratto added CHANGELOG entries for Prometheus, Cortex, and Loki.